### PR TITLE
Remove core dim in cpu total stats

### PR DESCRIPTION
**Pull request by dichiarafrancesco**

### CODE REVIEW

File: src/diamond/collectors/cpu/cpu.py [hunk 1]
```diff
@@ -185,6 +185,10 @@ def cpu_delta_time(interval):
 
             for metric_name in metrics.keys():
                 metric_value = metrics[metric_name]
+                if (str_to_bool(self.config['enableAggregation']) is False
+                    and ('user_mode' in metric_name
```
Comment (gnufied) [inline src/diamond/collectors/cpu/cpu.py]: can we write it like `metric_name in ['user_mode', 'irq_softirq']`  it reads better.

Comment (dichiarafrancesco) [inline src/diamond/collectors/cpu/cpu.py]: nop. i don't think it's the same think. in your case `metric_name` must be `user_mode` or `irq_softirq`.

**Comment by gnufied** [comment]
:shipit:

**Comment by gnufied** [comment]
:shipit:

**Pull request merged**